### PR TITLE
infra: sync template files changes from rhel-10 branch with master

### PR DESCRIPTION
**Pull request by KKoukiou**

### CODE REVIEW

File: dockerfile/anaconda-iso-creator/lorax-build [hunk 1]
```diff
@@ -42,10 +42,6 @@ createrepo_c $REPO_DIR
 # The --volid argument can cause different network interface naming: https://github.com/rhinstaller/kickstart-tests/issues/448
 lorax -p Fedora -v "$VERSION_ID" -r "$VERSION_ID" \
       --volid Fedora-S-dvd-x86_64-rawh \
-      -s http://dl.fedoraproject.org/pub/fedora/linux/development/rawhide/Everything/x86_64/os/ \
-      -s file://$REPO_DIR/ \
-      "$@" \
-      output || cp *.log "$OUT_DIR"
```
Comment (jkonecny12) [inline dockerfile/anaconda-iso-creator/lorax-build:48]: This one have to stay.

File: dockerfile/anaconda-iso-creator/lorax-build-webui.j2 [hunk 2]
```diff
@@ -20,6 +20,12 @@
 # /images (Where the boot.iso will be stored)
 #
 
+{% if distro_name == "rhel" %}
+#
+# WARNING: this solution is not supported on RHEL/CentOS Stream
+#
+{% endif %}
+
```
Comment (jkonecny12) [inline dockerfile/anaconda-iso-creator/lorax-build-webui.j2:28]: I would remove this or the above newline. In the rendered file it looks weird.

File: dockerfile/anaconda-iso-creator/lorax-build.j2 [hunk 3]
```diff
@@ -33,16 +33,29 @@ createrepo_c $REPO_DIR
...
+MAJOR_VERSION=${VERSION_ID%%.*}
+MINOR_VERSION=${VERSION_ID#*.}
+
+# FIXME: Remove COPR repository which is temporary workaround until python3-pam package is available
+lorax -p RHEL -v "$MAJOR_VERSION" -r "$MINOR_VERSION" \
+      --volid "RHEL-$MAJOR_VERSION-$MINOR_VERSION-BaseOS-x86_64" \
+      --nomacboot \
+      -s http://download.devel.redhat.com/rhel-{$ distro_release $}/nightly/RHEL-{$ distro_release $}-Public-Beta/latest-RHEL-{$ distro_release $}/compose/BaseOS/x86_64/os/ \
+      -s http://download.devel.redhat.com/rhel-{$ distro_release $}/nightly/RHEL-{$ distro_release $}-Public-Beta/latest-RHEL-{$ distro_release $}/compose/AppStream/x86_64/os/ \
+      -s https://download.copr.fedorainfracloud.org/results/m4rtink/python-pam-rebuild/centos-stream-10-x86_64/ \
```
Comment (jkonecny12) [inline dockerfile/anaconda-iso-creator/lorax-build.j2:54]: This is not correct. These are changes required only on RHEL-10. See Public-Beta.

File: dockerfile/anaconda-live-iso-creator/Dockerfile.j2 [hunk 4]
```diff
@@ -16,6 +16,12 @@
 #   repositories (VPN for example)
 #
 
+{% if distro_name == "rhel" %}
+#
+# WARNING: this solution is not supported on RHEL/CentOS Stream
+#
+{% endif %}
+
```
Comment (jkonecny12) [inline dockerfile/anaconda-live-iso-creator/Dockerfile.j2:24]: Same here.

File: dockerfile/anaconda-live-iso-creator/lmc-build.j2 [hunk 5]
```diff
@@ -13,6 +13,12 @@
 # Output directory:
 # /images (Where the boot.iso will be stored)
 
+{% if distro_name == "rhel" %}
+#
+# WARNING: this solution is not supported on RHEL/CentOS Stream
+#
+{% endif %}
+
```
Comment (jkonecny12) [inline dockerfile/anaconda-live-iso-creator/lmc-build.j2:21]: And here.

**Review by jkonecny12 — CHANGES_REQUESTED** [review]
This PR would break things. Please see the comments below.

File: dockerfile/anaconda-iso-creator/lorax-build (hunk 1 above)
Comment (KKoukiou) [reply dockerfile/anaconda-iso-creator/lorax-build]: Oops sorry, put back

File: dockerfile/anaconda-live-iso-creator/lmc-build.j2 (hunk 5 above)
Comment (KKoukiou) [reply dockerfile/anaconda-live-iso-creator/lmc-build.j2:20]: Yeap

File: dockerfile/anaconda-iso-creator/lorax-build (hunk 1 above)
Comment (KKoukiou) [reply dockerfile/anaconda-iso-creator/lorax-build]: Yeap sorry that was stupid.

File: dockerfile/anaconda-iso-creator/lorax-build.j2 (hunk 3 above)
Comment (KKoukiou) [reply dockerfile/anaconda-iso-creator/lorax-build.j2:54]: Yeap sorry that was stupid.

**Review by jkonecny12 — APPROVED** [review]
Looks great to me now

**Pull request merged**